styles/wallet.module.css

```diff
@@ -1,10 +1,13 @@
 .walletGrid {
   display: grid;
-  grid-template-columns: repeat(auto-fit, minmax(160px, 1fr));
+  grid-template-columns: repeat(auto-fill, 160px);
   grid-gap: 20px;
-  justify-items: center;
-  align-items: center;
+}
+
+.walletFilters {
   margin-top: 3rem;
+  margin-left: 0.2rem;
+  margin-bottom: 0.5rem;
```

Comment:
selecting and deselecting a filter quickly causes the word to be selected by the browser
![image](https://github.com/user-attachments/assets/91587aa0-9621-42c2-a317-09b568f949af)

maybe we should add
```
  user-select: none;
```
here